internal/scorecard/tests/basic.go

```diff
@@ -50,8 +50,7 @@ func checkSpec(crSet []unstructured.Unstructured,
 	res scapiv1alpha3.TestResult) scapiv1alpha3.TestResult {
 	for _, cr := range crSet {
 		if cr.Object["spec"] == nil {
-			res.Errors = append(res.Errors, fmt.Sprintf("error spec does not exist for the custom resource %s", cr.GetName()))
-			res.State = scapiv1alpha3.FailState
+			res.State = scapiv1alpha3.PassState
 			res.Suggestions = append(res.Suggestions, fmt.Sprintf("spec missing from [%+v]", cr.GetName()))
```

Comment:
Does suggestions == warning? If not, we should ensure we are making this a visible warning.

Reply:
I agree, updated it to `Warning`

FYI

```
	// Warnings pertain to issues with the manifest that are optional to correct.
	Warnings []string `json:"warnings,omitempty"`
	// Suggestions is a list of suggestions for the user to improve their score (if applicable)
	Suggestions []string `json:"suggestions,omitempty"`
```

Reply:
As discussed, I am keeping it as a `suggestion` only.